Python/Product/PythonTools/PythonTools/Environments/AddVirtualEnvironmentControl.xaml

```diff
@@ -142,10 +142,13 @@
                                     Converter={StaticResource ScreenAdjustingConverter}, 
                                     ConverterParameter=249}"
                                 HorizontalAlignment="Left"
-                                Text="{Binding Path=VirtualEnvName,Mode=TwoWay,UpdateSourceTrigger=PropertyChanged,ValidatesOnNotifyDataErrors=False}"
                                 AutomationProperties.AutomationId="EnvName"
                                 AutomationProperties.IsRequiredForForm="True"
-                                AutomationProperties.LabeledBy="{Binding ElementName=EnvNameLabel}"/>
```

Comment:
What does the changes in the file do?

Reply:
Binding modes are TwoWay by default, so I removed that. The main problem here is that `ValidatesOnNotifyDataErrors` was set to false, meaning this textbox won't validate when the codebehind sets errors on the `VirtualEnvName`.

This defaults to true, so I removed it. Now any errors set on `VirtualEnvName` will show up in the UI next to this textBox.